### ISSUE TITLE
Editor_re:Invent_Bug - Pressing Create Entity in the RMB context menu in the viewport parents the entity to the current selection.

### ISSUE THREAD

**Issue by AMZN-daimini** [o3de/o3de#5290]
**Describe the bug**
Recently, the "Create Entity" and "Create Child Entity" context menu items have been consolidated into a single, contextual "Create Entity" menu item. In the Outliner, right clicking an empty spot and creating a new entity will parent the new entity to the level container (or the container of the currently focused prefab), while right clicking an entity and creating a new entity will parent it to the one that was right-clicked. This works because right-clicking also changes the selection in the Outliner.

In the viewport, right-clicking in an empty spot does not change the current selection. That means that selecting the "Create Entity" menu item will create the entity under the current selection, which may not be as intuitive.

Before investigating the bug, this issue should be raised to the UX group to identify whether the current behavior is acceptable or we would want to rectify it (and how).

**Steps to reproduce**
1. Open a level
2. Select an entity
3. Right click in a random spot in the viewport
4. In the context menu, select "Create Entity"

**Expected behavior**
An entity is created at the spot that was right-clicked, and it's parented to the container of the currently focused prefab (or the level).

**Actual behavior**
The new entity is parented to the selection.

**Comment by alexmontAmazon** [comment]
yep, we just need confirmation from UX as to the preferred behavior here. @yuyihsu , could you help?
